remote/inc/block-patterns.php

```diff
@@ -42,6 +42,8 @@ function remote_register_block_patterns() {
 
 	$block_patterns = array(
 		'hero-text',
+		'posts-list',
+		'search-form',
```

Comment:
Just a quick note: I recommend when we hide a pattern that we also prefix the filename with `hidden-` to make it 100% clear without needing to look in the file. It's easy to miss that little `'inserter'` line. 🙂